### PR TITLE
Clarify MAX_SUBSCRIBE_ID

### LINKED ISSUE #547: Draft-06. MAX_SUBSCRIBE_ID error handling is a bit ambiguous
In 6.16 of draft 6, the draft says that "receipt of a MAX_SUBSCRIBE_ID message with an equal or smaller Subscribe ID value is a 'Protocol Violation'." ; however, later the section mentions that "If a Subscribe ID equal or larger than this is received in any message, including SUBSCRIBE, the publisher MUST close the session with an error of 'Too Many Subscribes'."

This is can be ambiguous because technically MAX_SUBSCRIBE_ID is any message. Should all sessions be closed if MAX_SUBSCRIBE_ID message contains an invalid Subscribe ID value? Does any message exclude MAX_SUBSCRIBE_ID if an invalid Subscribe ID value is provided? Clarification on this would be helpful. 

Thank you!

### PR DESCRIPTION
**Pull request by ianswett**
This is an attempt at clarifying MAX_SUBSCRIBE_ID, feel free to make suggestions.

Fixes #547

### CODE REVIEW

File: draft-ietf-moq-transport.md [hunk 1]
```diff
@@ -1839,9 +1839,9 @@ MAX_SUBSCRIBE_ID
 {: #moq-transport-max-subscribe-id format title="MOQT MAX_SUBSCRIBE_ID Message"}
 
 * Subscribe ID: The new Maximum Subscribe ID for the session. If a Subscribe ID
-equal or larger than this is received in any message, including SUBSCRIBE,
-the publisher MUST close the session with an error of 'Too Many Subscribes'.
-More on Subscribe ID in {{message-subscribe-req}}.
+equal or larger than this is received by the publisher that sent the
+MAX_SUBSCRIBE_ID, the publisher MUST close the session with an error of
+'Too Many Subscribes'. More on Subscribe ID in {{message-subscribe-req}}.
```
Comment (afrind) [inline draft-ietf-moq-transport.md:1844]: What you have is fine, but "More on Subscribe ID in ..." is awkward.  Just add a section reference on line 1841 after "Subscribe ID"

**Review by afrind — COMMENTED** [review]
lgtm

**Review by suhasHere — APPROVED** [review]
Agree with Alan's comment. Otherwise LGTM

**Review by fluffy — APPROVED** [review]

File: draft-ietf-moq-transport.md (hunk 1 above)
Comment (ianswett) [reply draft-ietf-moq-transport.md]: Done, PTAL

**Comment by afrind** [comment]
lgtm

**Pull request merged**